delegates.rb

```diff
@@ -257,13 +262,45 @@ def fetch(path, ip)
     http.use_ssl = true
 
     response = http.request(request)
-
     return response.body
   end
 
   def api_url(path)
     "https://api.repo.nypl.org/api/v2/#{path}"
   end
+  
+  def homepage_image_rights_hash

```

Comment:
Making this a frozen constant will make it a bit more performant.

Reply:
Totally great call.

Reply:
updated